dns/index.ts

```diff
@@ -12,6 +12,15 @@ export async function initDnsServer (port: number, store: SiteConfigStore, logge
       const [{ name }] = request.questions
       logger?.info(`[dns] ${rinfo.address}:${rinfo.port} asked for ${name}`)
 
+      // TODO: Pass server from config
+      response.authorities.push({
+        name,
+        type: dns2.Packet.TYPE.NS,
+        class: dns2.Packet.CLASS.IN,
+        ttl: TTL,
+        ns: 'api.distributed.press.'
```

Comment:
You can get the [host](https://github.com/hyphacoop/api.distributed.press/blob/main/index.ts#L27) param in [apiBuilder](https://github.com/hyphacoop/api.distributed.press/blob/main/api/index.ts#L46) and should pass it to the [DNS init](https://github.com/hyphacoop/api.distributed.press/blob/main/api/index.ts#L73) defined [here](https://github.com/hyphacoop/api.distributed.press/blob/main/dns/index.ts#L7)

Maybe respond to CNAME with the host too?

Reply:
> Maybe respond to CNAME with the host too?

it should query A and AAAA addresses for the host and respond with that, because CNAME is frowned upon on the apex domain (some implementations ignore it but on the wild i've only found it on email notifications from twilio)